spec/features/happy_journeys_spec.rb

```diff
@@ -458,6 +458,141 @@
     expect(page).to be_axe_clean
   end
 
+  scenario "registration journey when outside of catchment area (crown dependencies)" do
+    visit "/"
+    expect(page).to have_text("Before you start")
+    page.click_link("Start now")
+
+    expect(page).to be_axe_clean
+    expect(page).to have_text("Have you agreed a start date of")
+    page.choose("Yes", visible: :all)
+    page.click_button("Continue")
+
+    expect(page).to be_axe_clean
+    expect(page).to have_text("Have you already chosen an NPQ and provider?")
+    page.choose("Yes, I have chosen my NPQ and provider", visible: :all)
+    page.click_button("Continue")
+
+    # expect(page).to be_axe_clean
+    # TODO: aria-expanded
+    expect(page.current_path).to eql("/registration/teacher-catchment")
+    page.choose("Jersey, Guernsey or the Isle of Man", visible: :all)
```

Comment:
This is the key part of this test, choosing this and then seeing that you go down the same path as when you choose Scotland/Wales/Northern Ireland/Another Country.